Makefile

```diff
@@ -5,6 +5,7 @@ black = black -S -l 120 --target-version py38 pydantic tests
 .PHONY: install-linting
 install-linting:
 	pip install -r tests/requirements-linting.txt
+	mypy --install-types
```

Comment:
What stubs are missing? If I remember correctly this installs missing stubs caught in the last run. Here we didn't even run mypy. Are you sure it's necessary?

Reply:
you know, I think that was for VSCode's `mypy` runs to stop complaining (on `toml` as I recall). The CLI runs seem perfectly happy without it in a clean environment, so I'll remove it.

Reply:
actually, yep `make mypy` complains about the `toml` stubs.  Lighter touch to just go ahead and add that specifically to the dev requirements.

Reply:
done in https://github.com/samuelcolvin/pydantic/pull/2908/commits/067d7880402d1aa0f5ba673a17f135a00d327162